kong/clustering/control_plane.lua

```diff
@@ -180,7 +181,7 @@ end
 _M._update_compatible_payload = update_compatible_payload
 
 function _M:export_deflated_reconfigure_payload()
-  ngx_log(ngx_DEBUG, _log_prefix, "exporting config for legacy protocol")
+  ngx_log(ngx_DEBUG, _log_prefix, "exporting config")
```

Comment:
note for reviewer: `_log_prefix` is unique from `control_plane.lua` to `wrpc_control_plane.lua`, so the extra verbiage wasn't needed here (and it makes our test assertions more readable without it).